service_objects/fields.py

```diff
@@ -208,3 +208,37 @@ def clean(self, values):
             self.check_type(value)
             self.check_unsaved(value)
         return values
+
+
+class DictField(forms.Field):
+    """
+    A field for :class:`Service` that accepts a dictionary:
+
+        class PDFGenerate(Service):
+            context = DictField()
+
+            process(self):
+                context = self.cleaned_data['context']
+
+        PDFGenerate.execute({
+            'context': {'a': 1, 'b': 2}
+        })
+
+
+    """
+    error_required = "Input is required. Expected dictionary but got %(value)r."
+    error_type = "Input needs to be of type dict."
```

Comment:
These two error lines, please call `_()` on them for translations

Reply:
Fixed.